firebase/functions/src/index.ts

```diff
@@ -194,6 +193,38 @@ export const addUserToChain = functions
     }
   });
 
+const notifyChainAdmin = async (chainId: string, newUserId: string) => {
+  const chainData = await db.collection("chains").doc(chainId).get();
+  const chainAdminUid = await chainData.get("chainAdmin");
+
+  const adminUserAuthData = await admin.auth().getUser(chainAdminUid);
```

Comment:
Suggestion: use "chainAdmin" as variable name. I'd only use the longer style if you had fetched admin data from more places. Only use longer names if it makes things clearer.